jekyll/_includes/global-footer.html

```diff
@@ -1,101 +1,26 @@
-<footer class="footer" data-analytics="clicked-footer" data-analytics-parent>
```

Comment:
The Japanese version of `global-footer.html` will also need to be updated with these same changes.

Probably the quickest way is just to make a copy of the previous Japanese version, paste this English version over the existing file, and then copy individual strings from the backup copy into the new markup.

Reply:
Sure - added.